utils/getClaimStatus.tsx

```diff
@@ -7,66 +7,115 @@ import { ClaimStatusContent, I18nString, TextOptionalLink, TransLineProps } from
 import { ScenarioType } from './getScenarioContent'
 import getUrl, { UrlType } from './getUrl'
 
+type StepType = 'your-next-steps' | 'edd-next-steps'
+
+interface ClaimStatusScenarioJson {
+  heading: string
+  summary: TextOptionalLink
+  'your-next-steps': TextOptionalLink[]
+  'edd-next-steps': TextOptionalLink[]
+}
+
+/**
+ * Convert a ScenarioType into the string key used in json translation files.
+ */
+export function scenarioToString(scenarioType: ScenarioType): string {
+  return ScenarioType[scenarioType].toLowerCase()
+}
+
 /**
  * Helper function to get shared Claim Status translation string prefix.
  */
-function getTranslationPrefix(scenarioType: ScenarioType): I18nString {
-  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}`
+function buildTranslationPrefix(scenarioType: ScenarioType): I18nString {
+  return `claim-status:scenarios.${scenarioToString(scenarioType)}`
 }
 
 /**
  * Get Claim Status heading.
  */
-export function getClaimStatusHeading(scenarioType: ScenarioType): I18nString {
-  return getTranslationPrefix(scenarioType) + '.heading'
+export function buildClaimStatusHeading(scenarioType: ScenarioType): I18nString {
+  return buildTranslationPrefix(scenarioType) + '.heading'
 }
 
 /**
- * Get Claim Status summary.
- *
- * Note: This function has a lot of verbosity due to Typescript + eslint.
+ * Build a list of urls from keys in the json translation files.
  */
-export function getClaimStatusSummary(scenarioType: ScenarioType): TransLineProps {
-  const transLineProps: TransLineProps = {
-    i18nKey: getTranslationPrefix(scenarioType) + '.summary.text',
+export function buildTransLineLinks(linkKeys: string[] | undefined): string[] {
+  const links: string[] = []
+  if (linkKeys) {
+    if (linkKeys && linkKeys.length > 0) {
+      for (const linkKey of linkKeys) {
+        // Explicitly cast to one of the allowed keys in urls.json
+        const key = linkKey as UrlType
+        const url = getUrl(key)
+        if (url) {
```

Comment:
Well, it shouldn't happen, but it might very well happen if someone makes a typo in `claim-status.json`. I don't think we should throw an error because I think the end-user should still see all the rest of the content, even if one of the links is missing, but I do think we should probably log it and get some kind of alert or something. I'll add some logging to it.